client-course-schedulizer/src/utilities/helpers/readCSV.ts

```diff
@@ -294,126 +97,42 @@ export const csvStringToSchedule = (csvString: string): di.Schedule => {
-        case "prefix": {
-          course.prefixes = value.replace(" ", "").split(/[;,]/);
-          break;
-        }
-        case "number": {
-          course.number = value;
-          break;
-        }
-        case "section": {
-          sss.letter = value;
-          break;
-        }
-        case "studentHours": {
-          course.studentHours = Number(value);
-          break;
-        }
-        case "facultyHours": {
-          course.facultyHours = Number(value);
-          break;
-        }
-        case "startTimeStr":
-        case "startTime": {
-          firstMeeting.startTime = startTimeCase(value);
-          break;
-        }
-        case "duration": {
-          firstMeeting.duration = Number(value);
-          break;
-        }
-        case "location": {
-          [firstMeeting.location.building, firstMeeting.location.roomNumber] = locationCase(value);
-          break;
-        }
-        case "roomCapacity": {
-          firstMeeting.location.roomCapacity = Number(value);
-          break;
-        }
-        case "year": {
-          sss.year = Number.isInteger(Number(value)) ? Number(value) : value;
-          break;
-        }
-        case "term": {
-          sss.term = termCase(value);
-          break;
-        }
-        case "semesterLength":
-        case "half": {
-          sss.semesterLength = semesterLengthCase(value);
-          break;
-        }
-        case "days": {
-          firstMeeting.days = daysCase(value);
-          break;
-        }
-        case "globalMax": {
-          sss.globalMax = Number(value);
-          break;
-        }
-        case "localMax": {
-          sss.localMax = Number(value);
-          break;
-        }
-        case "anticipatedSize": {
-          sss.anticipatedSize = Number(value);
-          break;
-        }
-        case "instructors":
-        case "instructor": {
-          sss.instructors = instructorCase(value);
-          break;
-        }
-        case "comments": {
-          sss.comments = value;
-          break;
-        }
-        default: {
-          break;
-        }
-      }
-    });
-
-    // Check if the meeting is empty, and should be removed
-    if (firstMeeting.days === [] || firstMeeting.duration === 0) {
-      sss.meetings = [];
-    }
-
-    // Create a section for this row of the CSV, and add it to the schedule
-    const section: di.Section = {
-      // TODO: Allow for multiple meetings
-      ...sss,
-    };
+      });
 
-    // Check if there is already a course in the schedule with the same prefix and number
-    const existingCourse: di.Course[] = schedule.courses.filter((c) => {
-      return (
-        c.prefixes.every((p) => {
-          return course.prefixes.includes(p);
-        }) && c.number === course.number
-      );
-    });
+      // Check if the meeting is empty, and should be removed
+      // if (firstMeeting.days === [] || firstMeeting.duration === 0) {
+      //   section.meetings = [];
+      // }
```

Comment:
Does this need to be removed or un-commented?

Reply:
No. It will need to be added back once we can parse duration from `math-schedule.csv`. I was struggling with that so I think it should be in a future PR

Reply:
I guess I'll add it back so we don't forget later though